src/cli/index.ts

```diff
@@ -24,12 +24,12 @@ commander
   .option('-x, --extra <extra>', 'Name or relative path of the file with the custom extra types')
 
   // Define the action of the CLI
-  .action(({input, output, classesFile: extra}: {[key: string]: string | void}) => {
+  .action(({input, output, extra}: {[key: string]: string | void}) => {
     const isConfigFileFound: boolean = fs.existsSync('./tailwind.config.js');
     // If the config file is found or provided explicitly by the user...
     if (isConfigFileFound || !!input) {
       // Generate the types and write them to a file on disk
-      void new GeneratedFileWriter({
+      return new GeneratedFileWriter({
```

Comment:
I added a return of the promise so that error logs have a chance to print before program execution is terminated.